examples/with-jest/jest.config.js

```diff
@@ -7,14 +7,14 @@ module.exports = {
   setupFilesAfterEnv: ['<rootDir>/setupTests.js'],
   testPathIgnorePatterns: ['/node_modules/', '/.next/'],
   transform: {
-    '^.+\\.(js|jsx|ts|tsx)$': '<rootDir>/node_modules/babel-jest',
+    '^.+\\.(js|jsx|ts|tsx)$': require.resolve('babel-jest'),
     '^.+\\.css$': '<rootDir>/config/jest/cssTransform.js',
   },
   transformIgnorePatterns: [
     '/node_modules/',
     '^.+\\.module\\.(css|sass|scss)$',
   ],
   moduleNameMapper: {
-    '^.+\\.module\\.(css|sass|scss)$': 'identity-obj-proxy',
+    '^.+\\.module\\.(css|sass|scss)$': require.resolve('identity-obj-proxy'),
```

Comment:
Hmm, is this not working before? Why change this?

Reply:
It does but if it ends up matching files in external modules it will have to rely on hoisting to locate it, resolving it up front prevents that

Reply:
Is the Jest recommended way to do this?